### PR TITLE
v0.32.4

### PR DESCRIPTION
**Pull request by adfarth**
See PRs:
fix for rates with non standard units #236 
Update to PVWatts v8 from v6 #233 
Update test values due to PVWatts v8 Update #242 
Generator installed_cost_per_kw default #238

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,23 @@ Classify the change according to the following categories:
...
     ### Removed
 
+## Develop
+### Changed
+- Changed unit test expected values due to update to PVWatts v8, which slightly changed expected PV production factors.
+- Changed **fuel_avail_gal** default to 1e9 for on-grid scenarios (same as off-grid)
+### Fixed
+- Corrected `Generator` **installed_cost_per_kw** from 500 to 650 if **only_runs_during_grid_outage** is _true_ or 800 if _false_
+
+## v0.32.4
```
Comment (hdunham) [inline CHANGELOG.md:33]: This stuff under 0.32.4 hasn't been merged into master and released yet right? Need to combine with what's under Develop above.

File: src/core/pv.jl [hunk 2]
```diff
@@ -82,7 +82,7 @@
     If `azimuth` is not provided, then it is set to 180 if the site is in the northern hemisphere and 0 if in the southern hemisphere.
 
 """
-struct PV <: AbstractTech
+mutable struct PV <: AbstractTech
```
Comment (hdunham) [inline src/core/pv.jl:85]: fine for now but we should be trying to avoid making structs mutable

File: src/core/urdb.jl [hunk 3]
```diff
@@ -292,10 +295,12 @@ function parse_urdb_energy_costs(d::Dict, year::Int; time_steps_per_hour=1, bigM
                     end
                     if non_kwh_units
                         rate = rate_average
```
Comment (hdunham) [inline src/core/urdb.jl:297]: just directly set total_rate instead of creating intermediate variable rate?

File: CHANGELOG.md (hunk 1 above)
Comment (hdunham) [reply CHANGELOG.md:26]: done

File: src/core/urdb.jl (hunk 3 above)
Comment (hdunham) [reply src/core/urdb.jl]: Made the change so you can just double check it

File: src/core/pv.jl (hunk 2 above)
Comment (adfarth) [reply src/core/pv.jl:85]: @hdunham do you want to make an issue for this?

File: src/core/urdb.jl [hunk 4]
```diff
@@ -290,12 +293,10 @@ function parse_urdb_energy_costs(d::Dict, year::Int; time_steps_per_hour=1, bigM
                     else
                         tier_use = tier
                     end
-                    if non_kwh_units
-                        rate = rate_average
-                    else
-                        rate = get(d["energyratestructure"][period][tier_use], "rate", 0)
-                    end
-                    total_rate = rate + get(d["energyratestructure"][period][tier_use], "adj", 0)
+                    total_rate = non_kwh_units ?
```
Comment (adfarth) [inline src/core/urdb.jl:296]: @hdunham I don't understand this change. When would `total_rate = non_kwh_units ? ` evaluate to true?

File: src/core/urdb.jl [hunk 5]
```diff
@@ -270,6 +270,9 @@ function parse_urdb_energy_costs(d::Dict, year::Int; time_steps_per_hour=1, bigM
 
         for month in range(1, stop=12)
             n_days = daysinmonth(Date(string(year) * "-" * string(month)))
+            if month == 2 && isleapyear(year)
```
[the remark this answers is not in the document]
Comment (hdunham) [reply src/core/urdb.jl:273]: Oh yes I was being mindless

**Pull request merged**